### PR TITLE
Update Android Easings

### PR DESCRIPTION
**Pull request by mattleibow**

### CODE REVIEW

File: Android/AndroidEasingFunctions/cgmanifest.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "Registrations":[ 
+    {
+      "Component": {
+        "Type": "Maven",
+        "Maven": {
+          "ArtifactId": "library",
+          "GroupId": "com.daimajia.easing",
+          "Version": "2.1"
```
Comment (Redth) [inline Android/AndroidEasingFunctions/cgmanifest.json]: NuGetId ?

File: Android/AndroidEasingFunctions/source/AndroidEasingFunctions/AndroidEasingFunctions.csproj [hunk 2]
```diff
@@ -1,70 +1,47 @@
...
   <ItemGroup>
-    <LibraryProjectZip Include="..\..\externals\AndroidEasingFunctions.aar">
-      <Link>Jars\AndroidEasingFunctions.aar</Link>
-    </LibraryProjectZip>
+    <LibraryProjectZip Include="..\..\externals\AndroidEasingFunctions.aar" Link="Jars\AndroidEasingFunctions.aar" />
   </ItemGroup>
+
   <ItemGroup>
-    <None Include="packages.config" />
+    <PackageReference Include="Xamarin.Android.Support.v7.AppCompat" Version="28.0.0.1" />
```
Comment (Redth) [inline Android/AndroidEasingFunctions/source/AndroidEasingFunctions/AndroidEasingFunctions.csproj]: Do we want PrivateAssets="none" here?

File: Android/AndroidEasingFunctions/cgmanifest.json (hunk 1 above)
Comment (mattleibow) [reply Android/AndroidEasingFunctions/cgmanifest.json]: Is this inside the Maven or the Component?

File: Android/AndroidEasingFunctions/source/AndroidEasingFunctions/AndroidEasingFunctions.csproj (hunk 2 above)
Comment (mattleibow) [reply Android/AndroidEasingFunctions/source/AndroidEasingFunctions/AndroidEasingFunctions.csproj]: Yes we do!

**Review by Redth — APPROVED** [review]

**Pull request merged**